solidus_stripe.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |s|
   # to the stripe gateway.
   #
   # This was resolved in v1.60, but we still need to skip 1.58 & 1.59.
-  s.add_dependency "activemerchant", "~> 1.48", "!= 1.58.0", "!= 1.59.0"
+  s.add_dependency "activemerchant", ">= 1.100" # includes "Stripe Payment Intents: Fix fallback for Store"
```

Comment:
I feel like the comment should be moved into the commit description, maybe with a specific commit?